src/python/pants/backend/java/compile/javac_test.py

```diff
@@ -28,9 +28,6 @@
 from pants.jvm.util_rules import rules as util_rules
 from pants.testutil.rule_runner import QueryRule, RuleRunner
 
-# TODO(#12293): Stabilize network flakiness.
-pytestmark = pytest.mark.skip
```

Comment:
I also disabled tests for javac_binary_test.py

Reply:
Done.  I scoped the skip in that file to only the test that exercises --jdk and actually expects bootstrapping to occur